### DISCUSSION TITLE
This user account is blocked. Contact registration@vercel.com for more information.

### DISCUSSION THREAD

**Discussion by vuteyrin** [vercel/community#64] [Help] [answered]
i cannot login to vercel website, using github i alway get this message "
This user account is blocked. Contact registration@vercel.com for more information."

**Comment by Pieparker** [comment, marked as answer]
Hey there! Please reach out to the team at the provided email address from your email address that is associated with your Vercel account. Thanks!

**Reply by henk9** [reply]
> Please reach out to the team at the provided email address from your email address that is associated with your Vercel account.

I find this very confusing. I have not (yet) signed up to Vercel with any email address, so when I click
"Continue with GitHub" at https://vercel.com/signup, how do I know what email address you think is
associated with my Vercel account? Is it the email address I have chosen to be my primary address 
at my GitHub account? Or what address otherwise?

Apparently, others are having the same problem as me, please see:
[This user account is blocked. Contact registration@vercel.com for more information.](
https://github.com/vercel/next.js/discussions/33365)

Thanks!

**Reply by swarnava** [reply]
@henk9 if you are using GitHub to login, it's likely linked with your GitHub email. If you reach out to registration@vercel.com, we can double check and confirm. Thanks!
